xml/depl_nodes.xml

```diff
@@ -3067,22 +3067,22 @@ host1:/srv/nfs/share1 /mnt/nfs/share1 -o rsize=8192,wsize=8192,timeo=14,intr
   </bridgehead>
 
   <para>
-   This plug-in requires to configure access to the &vmware; NSX service.
+   This plug-in requires configuring access to the &vmware; service.
```

Comment:
`This plug-in is required to configure access to the &vmware; service.` ?

Feels like the initial variation was missing bits

Reply:
This is the only change that still doesn't make sense to me. @dmpop second eyes, please? :)

Reply:
give this version a whirl. Thanks for sticking with it. No response from tech folks.